src/Ticket.php

```diff
@@ -5017,9 +5017,10 @@ public static function showCentralList($start, $status = "process", bool $showgr
                         $options['criteria'][0]['value']      = self::CLOSED;
                         $options['criteria'][0]['link']       = 'AND';
 
+                        $duration = Entity::getUsedConfig('inquest_config', $_SESSION['glpiactive_entity'], 'inquest_duration');
```

Comment:
> The problem is that this configuration may differ from one entity to another, and this link may display satisfactions from multiple entities at once. Depending on the entities configuration, the current proposal could filter surveys from a child entity that are still valid.

This comes from https://github.com/glpi-project/glpi/pull/17610 which corrects this case on MAIN